### PR TITLE
[release-v1.4] Perform `max.poll.interval.ms` considering `max.poll.records` (#2439)

### PR DESCRIPTION
**Pull request by pierDipi**
Cherry-pick of https://github.com/knative-sandbox/eventing-kafka-broker/commit/fe37776828e85c46524c7d82b5e8e54a789e9443

### CODE REVIEW

**Comment by pierDipi** [comment]
GitHub ...
```

+ local operator_dir=/tmp/serverless-operator
+ git clone --branch main https://github.com/openshift-knative/serverless-operator.git /tmp/serverless-operator
Cloning into '/tmp/serverless-operator'...
fatal: unable to access 'https://github.com/openshift-knative/serverless-operator.git/': The requested URL returned error: 429
+ export GOPATH=/tmp/go
```
/retest

File: data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/main/ConsumerVerticleFactoryImpl.java [hunk 1]
```diff
@@ -165,7 +165,8 @@ public AbstractVerticle get(final DataPlaneContract.Resource resource, final Dat
            egress.getEgressConfig() :
            resource.getEgressConfig();
 
-       consumerConfigs.put(ConsumerConfig.MAX_POLL_INTERVAL_MS_CONFIG, maxProcessingTimeMs(egressConfig));
+       final var maxProcessingTime = maxProcessingTimeMs(consumerConfigs, egressConfig);
```
Comment (matzew) [inline data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/main/ConsumerVerticleFactoryImpl.java:168]: why this extra `var` ? To improve readability?

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**